eng/CredScanSuppression.json

```diff
@@ -1,4 +1,29 @@
 {
-    "tool": "Credential Scanner",
-    "suppressions": []
+  "tool": "Credential Scanner",
+  "suppressions": [
+    {
+      "placeholder": "efQ8O5wj+98pO9uoDas4uhlPwdp5GHCRs9N2Da2EC/g=",
```

Comment:
@sima-zhu when you disable the general rules please double check to see if these suppressions are still needed or not and if not please clean them out.